### PR TITLE
Add CLI command to get and put JBrowse config files in collaboration server

### LINKED ISSUE #214: Multiple tracks to be centrally loaded
We have a use case where we would like to "pre-load" tracks for the users. This will be a mix of GFFs and BAMs. We would like them to appear as separate tracks, and perhaps even be configured to display in different colours etc. At the moment, the admin users can load only GFFs centrally by using "Import features"; however, all information gets collapsed into the same track. This would be such a useful feature. Thank you.

### PR DESCRIPTION
**Pull request by shashankbrgowda**
Fixes https://github.com/GMOD/Apollo3/issues/433, fixed #214 

Example:
```
apollo-cli % yarn dev jbrowse get-config --profile apollo > config.json

apollo-cli % jbrowse add-track https://ftp.ebi.ac.uk/pub/databases/havana/ngs_havana/CLS/updated_master_table/human/pacBioSII-Cshl-CapTrap_HpreCap_0+_Brain03Rep1.bam --indexFile https://ftp.ebi.ac.uk/pub/databases/havana/ngs_havana/CLS/updated_master_table/human/pacBioSII-Cshl-CapTrap_HpreCap_0+_Brain03Rep1.bam.bai --assemblyNames=66bb60e83db15a4a2aff224d
Added track with name "pacBioSII-Cshl-CapTrap_HpreCap_0+_Brain03Rep1" and trackId "pacBioSII-Cshl-CapTrap_HpreCap_0+_Brain03Rep1" to ./config.json

apollo-cli % yarn dev jbrowse set-config --profile apollo config.json
Jbrowse configuartion added successfully to apollo
```

### CODE REVIEW

File: packages/apollo-cli/src/commands/jbrowse/set-config.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import * as fs from 'node:fs'
+import {
+  ImportJBrowseConfigChange,
```
Comment (garrettjstevens) [inline packages/apollo-cli/src/commands/jbrowse/set-config.ts:3]:
```suggestion
import type {
  SerializedImportJBrowseConfigChange,
```

Import only the types here, we can't use the actual object in the CLI code due to dependency issues.

File: packages/apollo-cli/src/commands/jbrowse/set-config.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const access: { address: string; accessToken: string } =
+      await this.getAccess(flags['config-file'], flags.profile)
+    const filehandle = await fs.promises.open(flags['input-file'])
+    const fileContent = await filehandle.readFile({ encoding: 'utf8' })
+    await filehandle.close()
+
+    const change = new ImportJBrowseConfigChange({
+      typeName: 'ImportJBrowseConfigChange',
+      newJBrowseConfig: JSON.parse(fileContent) as JBrowseConfig,
+    })
```
Comment (garrettjstevens) [inline packages/apollo-cli/src/commands/jbrowse/set-config.ts:53]:
```suggestion
    const change: SerializedImportJBrowseConfigChange = {
      typeName: 'ImportJBrowseConfigChange',
      newJBrowseConfig: JSON.parse(fileContent) as JBrowseConfig,
    }
```

File: packages/apollo-cli/src/commands/jbrowse/set-config.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    },
+  ]
+
+  static flags = {
+    'input-file': Flags.string({
+      char: 'i',
+      description: 'Input jbrowse configuration file',
+      required: true,
+    }),
+  }
```
Comment (garrettjstevens) [inline packages/apollo-cli/src/commands/jbrowse/set-config.ts:35]: Can you make this a positional argument instead of a flag?

**Review by garrettjstevens — CHANGES_REQUESTED** [review]

File: packages/apollo-cli/src/commands/jbrowse/set-config.ts (hunk 3 above)
Comment (shashankbrgowda) [reply packages/apollo-cli/src/commands/jbrowse/set-config.ts]: done

**Review by garrettjstevens — APPROVED** [review]

**Pull request merged**